images/kubekins-e2e/variants.yaml

```diff
@@ -31,10 +31,16 @@ variants:
     K8S_RELEASE: stable
     BAZEL_VERSION: 3.4.1
     OLD_BAZEL_VERSION: 2.2.0
+  '1.27':
+    CONFIG: main
+    GO_VERSION: 1.20.2
+    K8S_RELEASE: latest-1.27
+    BAZEL_VERSION: 3.4.1
+    OLD_BAZEL_VERSION: 2.2.0
```

Comment:
we need to phase this out at some point. kubernetes and test-infra haven't needed it for O(years)

Reply:
Thank you for the reminder! Let's have a separately PR to address those together :)